compose.yaml

```diff
@@ -0,0 +1,37 @@
+services:
+  llamalang:
+    image: llamalang
+    build: ./
+    command: "/bin/sh /llamalang/run.sh"
+    environment:
+      IS_DEV: 1
```

Comment:
as its only for dev we can rename compose.yaml to dev.yaml or compose_dev.yaml to make it clear its only for development

Reply:
We won't be using docker compose for production and if we change the name it won't be picked up by docker compose automatically so IMO we should leave it as is